npm install fails with ERESOLVE overriding peer dependency for @mintplayer/ng-player-progress

**Describe the bug**
 When running `npm install`, the process fails with an `ERESOLVE overriding peer dependency` warning and an `ETARGET no matching version found` error for `@mintplayer/ng-player-progress@^17.3.0`.

**To Reproduce**
Steps to reproduce the behavior:
1. Run `npm install`
2. See error

**Expected behavior**
`npm install` should complete without errors.

**Error log**

`npm WARN ERESOLVE overriding peer dependency
npm WARN While resolving: @mintplayer/player-provider@17.3.0
npm WARN Found: @mintplayer/ng-player-progress@undefined
npm WARN node_modules/@mintplayer/ng-player-progress
npm WARN
npm WARN Could not resolve dependency:
npm WARN peer @mintplayer/ng-player-progress@"^17.3.0" from @mintplayer/player-provider@17.3.0
npm WARN node_modules/@mintplayer/player-provider
npm WARN   peer @mintplayer/player-provider@"^17.3.0" from @mintplayer/ng-video-player@17.3.9
npm WARN   node_modules/@mintplayer/ng-video-player
npm WARN   1 more (@mintplayer/video-player)
npm ERR! code ETARGET
npm ERR! notarget No matching version found for @mintplayer/ng-player-progress@^17.3.0.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.`

**Environment (please complete the following information):**
- Node.js version: [e.g. 18.20.2]
- npm version: [e.g. 10.5.0]

**Additional context**
Add any other context about the problem here. Initially, I discovered this bug while doing an `npx npm-check-updates` routine. I've noticed a bump from v17 to 17.3.0. After modifying the version to 17.3, and running another `npm install`, I get the following error
`npm ERR! code ETARGET
npm ERR! notarget No matching version found for @mintplayer/ng-vimeo-player@17.3.0.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.`

--- a/libs/mintplayer-facebook-player/package.json
+++ b/libs/mintplayer-facebook-player/package.json
@@ -2,7 +2,7 @@
   "name": "@mintplayer/facebook-player",
   "private": false,
   "type": "module",
-  "version": "17.3.0",
+  "version": "17.4.0",
   "repository": {
     "type": "git",
     "url": "https://github.com/MintPlayer/mintplayer-ng-video-player",
@@ -11,7 +11,7 @@
   "main": "./src/index.js",
   "typings": "./src/index.d.ts",
   "peerDependencies": {
-    "@mintplayer/player-provider": "^17.3.0",
+    "@mintplayer/player-provider": "^17.4.0",
     "@mintplayer/script-loader": "^1.2.0",
     "rxjs": "^7.4.0"
   },

--- a/libs/mintplayer-file-player/package.json
+++ b/libs/mintplayer-file-player/package.json
@@ -2,7 +2,7 @@
   "name": "@mintplayer/file-player",
   "private": false,
   "type": "module",
-  "version": "17.3.0",
+  "version": "17.4.0",
   "repository": {
     "type": "git",
     "url": "https://github.com/MintPlayer/mintplayer-ng-video-player",
@@ -11,8 +11,8 @@
   "main": "./src/index.js",
   "typings": "./src/index.d.ts",
   "peerDependencies": {
-    "@mintplayer/player-provider": "^17.3.0",
-    "@mintplayer/ng-canvas-resizer": "^17.3.0",
+    "@mintplayer/player-provider": "^17.4.0",
+    "@mintplayer/ng-canvas-resizer": "^17.4.0",
     "rxjs": "^7.4.0"
   },
   "dependencies": {

--- a/libs/mintplayer-mixcloud-player/package.json
+++ b/libs/mintplayer-mixcloud-player/package.json
@@ -2,7 +2,7 @@
   "name": "@mintplayer/mixcloud-player",
   "private": false,
   "type": "module",
-  "version": "17.3.0",
+  "version": "17.4.0",
   "repository": {
     "type": "git",
     "url": "https://github.com/MintPlayer/mintplayer-ng-video-player",
@@ -11,7 +11,7 @@
   "main": "./src/index.js",
   "typings": "./src/index.d.ts",
   "peerDependencies": {
-    "@mintplayer/player-provider": "^17.3.0",
+    "@mintplayer/player-provider": "^17.4.0",
     "rxjs": "^7.4.0"
   },
   "dependencies": {

--- a/libs/mintplayer-ng-canvas-resizer/package.json
+++ b/libs/mintplayer-ng-canvas-resizer/package.json
@@ -1,7 +1,7 @@
 {
   "name": "@mintplayer/ng-canvas-resizer",
   "private": false,
-  "version": "17.3.0",
+  "version": "17.4.0",
   "repository": {
     "type": "git",
     "url": "https://github.com/MintPlayer/mintplayer-ng-video-player",

--- a/libs/mintplayer-ng-video-player/package.json
+++ b/libs/mintplayer-ng-video-player/package.json
@@ -11,8 +11,8 @@
     "@angular/common": "^17.0.0",
     "@angular/core": "^17.0.0",
     "rxjs": "^7.4.0",
-    "@mintplayer/player-provider": "^17.3.0",
-    "@mintplayer/player-progress": "^17.3.0",
+    "@mintplayer/player-provider": "^17.4.0",
+    "@mintplayer/player-progress": "^17.4.0",
     "@mintplayer/video-player": "^17.3.0"
   },
   "dependencies": {

--- a/libs/mintplayer-player-progress/package.json
+++ b/libs/mintplayer-player-progress/package.json
@@ -2,7 +2,7 @@
   "name": "@mintplayer/player-progress",
   "private": false,
   "type": "module",
-  "version": "17.3.0",
+  "version": "17.4.0",
   "repository": {
     "type": "git",
     "url": "https://github.com/MintPlayer/mintplayer-ng-video-player",

--- a/libs/mintplayer-player-provider/package.json
+++ b/libs/mintplayer-player-provider/package.json
@@ -2,7 +2,7 @@
   "name": "@mintplayer/player-provider",
   "private": false,
   "type": "module",
-  "version": "17.3.0",
+  "version": "17.4.0",
   "repository": {
     "type": "git",
     "url": "https://github.com/MintPlayer/mintplayer-ng-video-player",
@@ -11,7 +11,6 @@
   "main": "./src/index.js",
   "typings": "./src/index.d.ts",
   "peerDependencies": {
-    "@mintplayer/ng-player-progress": "^17.3.0",
     "rxjs": "^7.4.0"
   },
   "dependencies": {

--- a/libs/mintplayer-playlist-controller/package.json
+++ b/libs/mintplayer-playlist-controller/package.json
@@ -2,7 +2,7 @@
   "name": "@mintplayer/playlist-controller",
   "private": false,
   "type": "module",
-  "version": "17.3.0",
+  "version": "17.4.0",
   "repository": {
     "type": "git",
     "url": "https://github.com/MintPlayer/mintplayer-ng-video-player",

--- a/libs/mintplayer-soundcloud-player/package.json
+++ b/libs/mintplayer-soundcloud-player/package.json
@@ -2,7 +2,7 @@
   "name": "@mintplayer/soundcloud-player",
   "private": false,
   "type": "module",
-  "version": "17.3.0",
+  "version": "17.4.0",
   "repository": {
     "type": "git",
     "url": "https://github.com/MintPlayer/mintplayer-ng-video-player",
@@ -11,7 +11,7 @@
   "main": "./src/index.js",
   "typings": "./src/index.d.ts",
   "peerDependencies": {
-    "@mintplayer/player-provider": "^17.3.0",
+    "@mintplayer/player-provider": "^17.4.0",
     "@mintplayer/script-loader": "^1.2.0",
     "rxjs": "^7.4.0"
   },

--- a/libs/mintplayer-spotify-player/package.json
+++ b/libs/mintplayer-spotify-player/package.json
@@ -2,7 +2,7 @@
   "name": "@mintplayer/spotify-player",
   "private": false,
   "type": "module",
-  "version": "17.3.0",
+  "version": "17.4.0",
   "repository": {
     "type": "git",
     "url": "https://github.com/MintPlayer/mintplayer-ng-video-player",
@@ -11,7 +11,7 @@
   "main": "./src/index.js",
   "typings": "./src/index.d.ts",
   "peerDependencies": {
-    "@mintplayer/player-provider": "^17.3.0",
+    "@mintplayer/player-provider": "^17.4.0",
     "@mintplayer/script-loader": "^1.2.0",
     "rxjs": "^7.4.0"
   },

--- a/libs/mintplayer-streamable-player/package.json
+++ b/libs/mintplayer-streamable-player/package.json
@@ -2,7 +2,7 @@
   "name": "@mintplayer/streamable-player",
   "private": false,
   "type": "module",
-  "version": "17.3.0",
+  "version": "17.4.0",
   "repository": {
     "type": "git",
     "url": "https://github.com/MintPlayer/mintplayer-ng-video-player",
@@ -11,7 +11,7 @@
   "main": "./src/index.js",
   "typings": "./src/index.d.ts",
   "peerDependencies": {
-    "@mintplayer/player-provider": "^17.3.0",
+    "@mintplayer/player-provider": "^17.4.0",
     "@mintplayer/script-loader": "^1.2.0",
     "rxjs": "^7.4.0"
   },

--- a/libs/mintplayer-twitch-player/package.json
+++ b/libs/mintplayer-twitch-player/package.json
@@ -2,7 +2,7 @@
   "name": "@mintplayer/twitch-player",
   "private": false,
   "type": "module",
-  "version": "17.3.0",
+  "version": "17.4.0",
   "repository": {
     "type": "git",
     "url": "https://github.com/MintPlayer/mintplayer-ng-video-player",
@@ -11,7 +11,7 @@
   "main": "./src/index.js",
   "typings": "./src/index.d.ts",
   "peerDependencies": {
-    "@mintplayer/player-provider": "^17.3.0",
+    "@mintplayer/player-provider": "^17.4.0",
     "@mintplayer/script-loader": "^1.2.0",
     "rxjs": "^7.4.0"
   },

--- a/libs/mintplayer-video-player/package.json
+++ b/libs/mintplayer-video-player/package.json
@@ -2,7 +2,7 @@
   "name": "@mintplayer/video-player",
   "private": false,
   "type": "module",
-  "version": "17.3.0",
+  "version": "17.4.0",
   "description": "Typescript video player supporting multiple platforms",
   "repository": {
     "type": "git",
@@ -14,7 +14,7 @@
   "dependencies": {
     "tslib": "^2.3.0",
     "rxjs": "^7.4.0",
-    "@mintplayer/player-progress": "^17.3.0",
+    "@mintplayer/player-progress": "^17.4.0",
     "@mintplayer/player-provider": "^17.3.0"
   },
   "author": {

--- a/libs/mintplayer-vidyard-player/package.json
+++ b/libs/mintplayer-vidyard-player/package.json
@@ -2,7 +2,7 @@
   "name": "@mintplayer/vidyard-player",
   "private": false,
   "type": "module",
-  "version": "17.3.0",
+  "version": "17.4.0",
   "repository": {
     "type": "git",
     "url": "https://github.com/MintPlayer/mintplayer-ng-video-player",
@@ -11,7 +11,7 @@
   "main": "./src/index.js",
   "typings": "./src/index.d.ts",
   "peerDependencies": {
-    "@mintplayer/player-provider": "^17.3.0",
+    "@mintplayer/player-provider": "^17.4.0",
     "@mintplayer/script-loader": "^1.2.0",
     "@vidyard/embed-code": "^4.0.0",
     "rxjs": "^7.4.0"

--- a/libs/mintplayer-vimeo-player/package.json
+++ b/libs/mintplayer-vimeo-player/package.json
@@ -2,7 +2,7 @@
   "name": "@mintplayer/vimeo-player",
   "private": false,
   "type": "module",
-  "version": "17.3.0",
+  "version": "17.4.0",
   "repository": {
     "type": "git",
     "url": "https://github.com/MintPlayer/mintplayer-ng-video-player",
@@ -11,7 +11,7 @@
   "main": "./src/index.js",
   "typings": "./src/index.d.ts",
   "peerDependencies": {
-    "@mintplayer/player-provider": "^17.3.0",
+    "@mintplayer/player-provider": "^17.4.0",
     "@mintplayer/script-loader": "^1.2.0",
     "rxjs": "^7.4.0"
   },

--- a/libs/mintplayer-wistia-player/package.json
+++ b/libs/mintplayer-wistia-player/package.json
@@ -2,7 +2,7 @@
   "name": "@mintplayer/wistia-player",
   "private": false,
   "type": "module",
-  "version": "17.3.0",
+  "version": "17.4.0",
   "repository": {
     "type": "git",
     "url": "https://github.com/MintPlayer/mintplayer-ng-video-player",
@@ -11,7 +11,7 @@
   "main": "./src/index.js",
   "typings": "./src/index.d.ts",
   "peerDependencies": {
-    "@mintplayer/player-provider": "^17.3.0",
+    "@mintplayer/player-provider": "^17.4.0",
     "@mintplayer/script-loader": "^1.2.0",
     "rxjs": "^7.4.0"
   },

--- a/libs/mintplayer-youtube-player/package.json
+++ b/libs/mintplayer-youtube-player/package.json
@@ -1,7 +1,7 @@
 {
   "name": "@mintplayer/youtube-player",
   "private": false,
-  "version": "17.3.0",
+  "version": "17.4.0",
   "repository": {
     "type": "git",
     "url": "https://github.com/MintPlayer/mintplayer-ng-video-player",
@@ -10,7 +10,7 @@
   "main": "./src/index.js",
   "typings": "./src/index.d.ts",
   "peerDependencies": {
-    "@mintplayer/player-provider": "^17.3.0",
+    "@mintplayer/player-provider": "^17.4.0",
     "@mintplayer/script-loader": "^1.2.0",
     "rxjs": "^7.4.0"
   },

